packages/manager/modules/pci/src/projects/project/training/dashboard/install/install.controller.js

```diff
@@ -0,0 +1,15 @@
+import { GUIDE_URL } from '../../training.constants';
+
+export default class PciTrainingInstallController {
+  $onInit() {
+    this.guideUrl = GUIDE_URL;
+    // eslint-disable-next-line prefer-destructuring
+    this.currentRegion = this.regions[0];
+    this.osInstall = {
+      Linux: `curl ${this.currentRegion.cliInstallUrl} | sh`,
+      'Mac OS': `curl ${this.currentRegion.cliInstallUrl} | sh`,
+      Windows: `curl ${this.currentRegion.cliInstallUrl} | sh`,
+    };
```

Comment:
Looks like it's the same for all OS. Has it any chance to change or maybe we can have one single constant ?

Reply:
for now, but will change. We don't have the final commands for now